docs/modules/ROOT/nav.adoc

```diff
@@ -16,4 +18,7 @@
 * xref:udc.adoc[Universal Deployer Contract]
 * xref:utilities.adoc[Utilities]
 
-* xref:contracts::index.adoc[Contracts for Solidity]
\ No newline at end of file
+* xref:contracts::index.adoc[Contracts for Solidity]
+
+* API
+** xref:/api/account.adoc[Account]
```

Comment:
I haven't moved API inside each section yet, waiting for the "Default Collapsed Nav Item" option @frangio said he would try to add. I've been thinking that it might be better to leave API where it is and for example add a small button in the header of each section redirecting to that, though.

Reply:
> I've been thinking that it might be better to leave API where it is 

:smiling_face_with_tear: https://github.com/OpenZeppelin/docs.openzeppelin.com/pull/369

Reply:
Moved already 💯